_posts/2021-07-06-Influential-tech-lead.md

```diff
@@ -0,0 +1,90 @@
+However, this can never be accomplished by power and control, but only by genuine commitment of other people.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2021-07-06-Influential-tech-lead/planned-behaviour.png' | prepend: site.baseurl }}" alt="Planned Behaviour" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+This simplified diagram is based on "The theory of planned behaviour" by Icek Ajzen, a social psychologist and professor emeritus at the University of Massachusetts Amherst.
+His theory assumes that before every Behaviour, there is an Intention.
+
+A simple practical example: you notice a specific behaviour from a software engineer on your team, namely that he refuses to write unit tests.
+As a technical team lead, you are interested in why he formed the intention not to write these unit tests because you're convinced that unit tests have a variety of benefits such as improving of code quality, finding bugs early and facilitating change.
+
+There are three elements that can help us trying to understand why your colleague formed this intention:
+
+- **Attitude a.k.a "Is the behaviour good and is it right?"**:  We're deciding if the behaviour is in our own best interest or if it is the right thing to do. So if we have a positive attitude towards the behaviour, it is more likely that we will perform the behaviour. 
+- **Subjective norm a.k.a "What's everyone else doing?"**: We're asking ourselves what others think of the behaviour and how they judge it. If we think that others consider the behaviour to be normal or good, it is more likely that we will perform the behaviour. 
+- **Perceived behavioural control a.k.a "Can I do it?"**: When we believe that behaviour is easy to perform, we are more likely to perform the behaviour.
+
+So when you're asking the software engineer on your team to write unit tests, the three elements listed above will cross his mind and he will ask himself questions like: "Is it a good thing to write unit tests?", "Is it in my own interest to write unit tests?", "Does everyone write unit tests or is it just me?", "Do I know how to write unit tests and do I have enough information/time to write them?", ...
+Targeting each and every one of these elements is the key to success when you want your colleague to start writing unit tests.
+
+So what can you do to influence the other person's thinking?
+
+- **"Is the behaviour good and is it right?"**: Identify what the other person cares about, what he thinks is good and what he thinks is the right thing to do. For example, your colleague might think it's super important to finish his user story as quickly as possible. The key thing to do here as a technical team lead is not to be manipulative or dishonest, but to genuinely align your goals with his motivations and things he already cares about. Writing unit tests facilitates change, provides documentation, and ultimately saves time.
```

Comment:
I would not use 'he' here. 
he/she, he or she , 'that person'

Reply:
ok